### DISCUSSION TITLE
Where can I find a simple, light-background theme?

### DISCUSSION THREAD

**Discussion by npanzica-qontigo** [KDE/ghostwriter#651] [Q&A]
I'm new to this editor. I like it. The side-by-side feature (code vs. formatted version) is very useful, but it did not come with dark and light themes (which was stated somewhere in the info about it). Instead, my version (2.0.2)  came with two dark themes. I'd like to find a very basic light-background theme, something very simple with a standard-looking, easy-to-read font, that I can install. Any recommendations?

**Comment by tildagail** [comment]
The dark themes are also light themes. If you click on the little moon in the bottom right corner*, Ghostwriter toggles between dark mode and light mode.

(* next to the toggle for html preview, hemingway mode, distraction free etc.)

Font is changed in the Preferences window, not as part of the theme, so you can use any font that's installed on your computer.
